ember.s

```diff
@@ -183,7 +192,29 @@ not_run:
 	jmp shell
 
 not_rw:
-	jmp short parse_error ; TODO: make this a direct jump when features are finalized
+	cmp al, 'p'
+	jnz short not_print
+
+	mov cx, 16
+	mov si, di
+	mov ah, 0x0e
+	mov bx, 0x0007
+.loop:	lodsb
+	int 0x10
+	loop .loop
+
+	mov al, 0x0d
+	int i_putchar
```

Comment:
What's the motivation for the print command? If it's to view memory as ASCII, perhaps it would be better to integrate it into the hexdump? Also, why not reuse `i_putchar`? Moreover, I feel like the read location would be a better choice here (see ["Inferred locations"](https://github.com/NieDzejkob/ember#inferred-locations) in the README).

Reply:
I didn't use i_putchar here because it prints 0x0d as 0x0a 0x0d